templates/security/cve/cve.html

```diff
@@ -137,11 +137,11 @@ <h2>Status</h2>
                   {% if release.codename in statuses %}
                     {% if loop.index == 1 %}
                       <td rowspan="{{ releases | length }}">
-                        <a href="/security/cve?q=&package={{ package_name }}" target="_blank">{{ package_name }}</a><br>
+                        <a href="/security/cve?q=&package={{ package_name }}" target="_blank" rel="noopener noreferrer">{{ package_name }}</a><br>
                         <small>
-                        <a href="https://launchpad.net/distros/ubuntu/+source/{{ package_name }}" target="_blank">Launchpad</a>,
-                        <a href="https://packages.ubuntu.com/search?suite=all&section=all&arch=any&searchon=sourcenames&keywords={{ package_name }}" target="_blank">Ubuntu</a>,
-                        <a href="https://tracker.debian.org/{{ package_name }}" target="_blank">Debian</a>
+                        <a href="https://launchpad.net/distros/ubuntu/+source/{{ package_name }}" target="_blank" rel="noopener noreferrer">Launchpad</a>,
+                        <a href="https://packages.ubuntu.com/search?suite=all&section=all&arch=any&searchon=sourcenames&keywords={{ package_name }}" target="_blank" rel="noopener noreferrer">Ubuntu</a>,
+                        <a href="https://tracker.debian.org/{{ package_name }}" target="_blank" rel="noopener noreferrer">Debian</a>
```

Comment:
Why would any of these be blank. Is there state the user would lose by navigating away at this point?